app/src/main/java/com/orange/ouds/app/ui/tokens/TokenCategoryDetailScreen.kt

```diff
@@ -75,9 +82,14 @@ import com.orange.ouds.foundation.utilities.BasicPreviewParameterProvider
 @OptIn(ExperimentalFoundationApi::class)
 @Composable
 fun TokenCategoryDetailScreen(tokenCategory: TokenCategory<*>, onSubcategoryClick: (Long) -> Unit) {
-
     Screen {
-        LazyColumn(contentPadding = PaddingValues(bottom = OudsTheme.spaces.fixed.medium)) {
+        LazyColumn(
+            modifier = Modifier.consumeWindowInsets(WindowInsets.statusBars.only(WindowInsetsSides.Top)),
+            contentPadding = PaddingValues(
+                top = WindowInsets.statusBars.asPaddingValues().calculateTopPadding() + TopAppBarDefaults.TopAppBarExpandedHeight,
+                bottom = OudsTheme.spaces.fixed.medium
+            )
+        ) {
```

Comment:
It is the same `modifier` and `contentPadding` than in `ComponentVariantsScreen.kt`. Should we factorize?